### PR TITLE
Add a method to retrieve the interfaces in a sandbox

### PR DESCRIPTION
**Pull request by huikang**
Signed-off-by: Hui Kang hkang.sunysb@gmail.com

### CODE REVIEW

**Comment by huikang** [comment]
@boucher @aboch 
Rebase #465 on the latest libnetwork and move the function to sandbox

**Comment by boucher** [comment]
@aboch @mavenugo any feedback on this proposal?

File: sandbox.go [hunk 1]
```diff
@@ -36,6 +36,8 @@ type Sandbox interface {
 	SetKey(key string) error
 	// Delete destroys this container after detaching it from all connected endpoints.
 	Delete() error
+	// Retrieve the interfaces in the sandbox, for restoring checkpointed container
+	Interfaces() []osl.Interface
```
Comment (mavenugo) [inline sandbox.go]: though `Interface` is exposed as an osl interface, it feels odd to include it as part of the API.
osl is purely an internal implementation that provides os layer abstraction away from the clients.
Having this interface exposed via Interfaces() API which gives more control over life-cycle of the interface such as `Remove` makes me uncomfortable.

Lets hear from @mrjana on his opinion.

**Comment by huikang** [comment]
@boucher @mavenugo I actually created a issue regarding how to checkpoint/restore the container network correct. #524 
There, I hope we can design the interface correctly and make some agreement. Thanks.

**Comment by mavenugo** [comment]
@huikang responded to #524.

**Pull request closed without merging**